### ISSUE TITLE
Dynamic JavaScript API Client Proxies not Working (Bug?)

### ISSUE THREAD

**Issue by biomade** [abpframework/abp#13300]
version: 5.3
interface: mvc
database: ef core

I am using the framework and I am trying to create my first list page based upon the BookStore tutorial. I have created my DTOs and services and tested returning my list (1 item) via Swagger. My page is in the Types folder and should return a list of "type" from the Types table.   _NOTE: I did the BookStore tutorial and I get a similar message._

In the index.js file is the following line:
`ajax: abp.libs.datatables.createAjax(tas.tooltracker.types.type.getList)`

When I run the code, my list is not returned and the browser has the following error:

jquery.js?_v=637932416006032627:4050 jQuery.Deferred exception: tas is not defined ReferenceError: tas is not defined at HTMLDocument. (https://localhost:44338/Pages/Types/Index.js?_v=637933201112904158:13:50)

My namespace for the page is "TAS.ToolTracker.Types"

After much reading I have found that the Dynamic JavaScript API Client Proxies is not working.
By generating the proxy and setting it up as static, my code now runs but but it would be nice to have them be dynamic.

**Comment by realLiangshiwei** [comment]
Hi,

What is the result when you visit to `/Abp/ServiceProxyScript`?

**Comment by biomade** [comment]
I am not at the computer but what do you mean by "visit" , do you mean in swagger?

**Comment by realLiangshiwei** [comment]
You can check the document: https://docs.abp.io/en/abp/latest/UI/AspNetCore/Dynamic-JavaScript-Proxies#service-proxy-script-endpoint

**Comment by biomade** [comment]
I commented out the disabling of the dynamic proxy, removed the loading of the the static proxy script as well as excluding it from the project just to be safe. I cleaned and rebuilt the solution then ran it. I entered the endpoint as suggested and I see the js file contents but the parameter for the datatables.createAjax is slighty different in casing. I have updated my index.js file to what what shown as a result of calling the end point and THAT FIXED IT. 

@realLiangshiwei  - thank you so much for your help with this. I shall now be able to get a good night's sleep and not have code running around in my head!
